apps/src/blockly/eventHandlers.js

```diff
@@ -1,3 +1,7 @@
+// Event Handlers for Google Blockly.
```

Comment:
Is it bad that I didn't know this file existed? There are some other event handlers that we could potentially move here if we like this structure.